### PR TITLE
Added the package collections launch blog post

### PR DESCRIPTION
**Pull request by daveverwer**

### CODE REVIEW

File: Content/posts/launching-swift-package-collections.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+What are package collections? JSON descriptions of Swift package metadata. What does the Swift Package Index have in droves? Metadata about Swift packages! As soon as we heard about [the proposal](https://forums.swift.org/t/se-0291-package-collections/41905), we knew we had to support it. For a comprehensive description of the feature, [check out the post over on the official Swift blog](https://swift.org/blog/package-collections/).
+
+We have big plans for package collections, but we wanted to ship something alongside this first beta release. We chose to implement owner collections, a list of all packages from the index written by a repository owner. So, you could get a package collection containing [all packages from Alamofire](https://swiftpackageindex.com/Alamofire) or [all packages from Point-Free](https://swiftpackageindex.com/pointfreeco).
```
Comment (finestructure) [inline Content/posts/launching-swift-package-collections.md:11]: That second sentence sounds a bit awkward. Maybe:

> So we implemented owner [author?] collections, which bundles an author’s or organisation’s complete set of packages into a collection.

(I’ll hit approve in a sec, up to you if you fancy amending!)

Comment (finestructure) [reply Content/posts/launching-swift-package-collections.md:11]: Oops, double “So” opening with that.

File: Content/posts/launching-swift-package-collections.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+So, we expected this feature to roll out as part of the Swift Package Manager command-line tool, and it did. What we didn't expect was full support for package collections in Xcode 13! What a pleasant surprise that was. 😍
```
Comment (finestructure) [inline Content/posts/launching-swift-package-collections.md:13]: Triple (Double) “So” :D

**Review by finestructure — APPROVED** [review]

File: Content/posts/launching-swift-package-collections.md (hunk 1 above)
Comment (daveverwer) [reply Content/posts/launching-swift-package-collections.md:11]: Went with:

> We chose to implement author collections, a list of all packages from the index grouped by repository owner.

**Pull request merged**